### ISSUE TITLE
#-247418147------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1845]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div id="attr-label-longitude" class="sc-dUWDJJ eBsayS" aria-describedby="calcite-tooltip-0f7318d4-c631-3272-7196-a5bbfb450a1d">longitude</div>`

<strong>Element Location: </strong>#attr-label-longitude

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-247418147_All page content should be contained by landmarks.png`
